helmclient.go

```diff
@@ -643,6 +703,12 @@ func (c *Client) newTunnel() (*k8sportforward.Tunnel, error) {
 		return nil, microerror.Mask(err)
 	}
 
+	// Do not create a tunnel if tiller is outdated. It will be upgraded.
```

Comment:
```suggestion
	// Do not create a tunnel if tiller is outdated.
```

You never know. There is no guarantee that `EnsureTillerInstalled` will be called.

In general I like the idea of not creating tunnel for incompatible tiller version. That solves the problem of talking with updated tiller transparently.

Reply:
Good point. Done.